opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/ba/AddChildFamilyMemberBa.java

```diff
@@ -0,0 +1,200 @@
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:gender")))
+                .perform(scrollTo(), click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Male"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:disabilities")))
+                .perform(scrollTo(), click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("No"))
+                .perform(click());
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:birth_cert_available")))
+                .perform(scrollTo(), click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Yes"))
+                .perform(click());
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:birth_regist_number")))
+                .perform(scrollTo(), typeText("234567890"));
+        Thread.sleep(500);
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:rhc_card")))
+                .perform(scrollTo(), click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("No"))
+                .perform(click());
+        onView(ViewMatchers.withSubstring("Save"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .check(matches(isDisplayed()));
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("MEMBERS"))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void addFamilyWithBlankFieldsBa() throws Throwable {
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.familyName), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .perform(click());
+        onView(withId(R.id.fab))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Add new family member"))
+                .perform(click());
+        Thread.sleep(100);
+        onView(ViewMatchers.withSubstring("Child under 5 years"))
+                .perform(click());
+        Thread.sleep(100);
+        Activity activity = getCurrentActivity();
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:dob")))
+                .perform(scrollTo(), click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("done")).perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Save"))
+                .perform(click());
+        Thread.sleep(100);
+        onView(ViewMatchers.withSubstring("Found 7 error(s) in the form. Please correct them to submit."))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void confirmUniqueIDPrepopulatedBa() throws Throwable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)